garage/tf/algos/off_policy_rl_algorithm.py

```diff
@@ -53,6 +53,31 @@ def log_diagnostics(self, paths):
         self.policy.log_diagnostics(paths)
         self.qf.log_diagnostics(paths)
 
+    def process_samples(self, itr, paths):
+        """Return processed sample data based on the collected paths.
+
+        Args:
+            itr(int): Iteration number.
+            paths(list): A list of collected paths.
```

Comment:
list of what?

Reply:
Don't really get your point here. Not "A list of collected paths"?

Reply:
when the type in a doctstring is a collection, we usually include the inner type. e.g. a list of strings is `list[str]`